play.pokemonshowdown.com/src/battle-log.ts

```diff
@@ -415,13 +415,61 @@ export class BattleLog {
 				this.messageFromLog('Just kidding! It was **Earthquake**!');
 				return true;
 			} else if (moveid === 'taunt') {
-				// April Fool's 2013
+				// April Fool's 2013, expanded in 2025
 				messageFromArgs(args, kwArgs);
 				const quotes = [
 					"Yo mama so fat, she 4x resists Ice- and Fire-type attacks!",
 					"Yo mama so ugly, Captivate raises her opponent's Special Attack!",
 					"Yo mama so dumb, she lowers her Special Attack when she uses Nasty Plot!",
-					"Yo mama so dumb, she thought Sylveon would be Light Type!",
+					"Yo mama so fat, she eats her Aguav Berry at 50% HP!",
+					"Yo Mama so stupid, she tried to count clouds one through eight!",
+					"Yo Mama so stupid, she thought Sticky Hold meant she needed to grab a stick!",
+					"Yo Mama so old, her Paradox forme would be Great Grandma!",
+					"Yo Mama so stupid, she can't learn anything from the move tutor!",
+					"Yo Mama so rude, she got Kyogre and Groudon to team up against her!",
+					"Yo Mama so fat, Focus Blast never misses her!",
+					"Yo Mama so fat, the Darkest Day is when she's tanning!",
+					"Yo Mama so stinky, her damaging moves have a 10% chance to make the opponent flinch!",
+					"Yo Mama so old, she has the NPCs from Legends Arceus in her yearbook!",
+					"Yo Mama so stupid, she's still trying to find Rhydon!",
+					"Yo Mama so stupid, she asked if the Insect Plate comes with an appetizer!",
+					"0 Atk Magikarp Splash vs. Lvl 1 0 HP / 0- Def Yo Mama : 321-378 (675 - 750%) -- guaranteed OHKO",
```

Comment:
Hm. I think the calcs may be unfamiliar to the most casual users? We probably have enough that we're not missing much by leaving them off.

I guess a lot of the jokes are incomprehensible if you're not familiar with the references, but the calcs are, like, an extra level of incomprehensible.

Reply:
removed the calcs